distribution/changelog.txt

```diff
@@ -14,6 +14,7 @@
 - Fix: [#15170] Plugin: incorrect label text alignment.
 - Fix: [#15184] Crash when hovering over water types in Object Selection.
 - Fix: [#15193] Crash when rides/stalls are demolished.
+- Fix: [#15255] Normal wall scenery would have the banner index 0 at all times.
```

Comment:
I'm not sure about the wording here, this is not some bug you get to see without looking hard enough with the tile inspector.

Reply:
I think it’s better to list the symptoms, which we almost always do anyway:

```suggestion
- Fix: [#15255] Tile Inspector shows banner information on walls that do not contain one.
```